### ISSUE TITLE
[Bug] "dbt deps" fails with ModuleNotFoundError: No module named 'pytz'

### ISSUE THREAD

**Issue by heiderich** [oracle/dbt-oracle#75]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

After installting dbt-oracle 1.4.0 I get the following error when running
`dbt deps`:

```
Traceback (most recent call last):
  File "/usr/local/bin/dbt", line 5, in <module>
    from dbt.main import main
  File "/usr/local/lib/python3.10/site-packages/dbt/main.py", line 24, in <module>
    import dbt.task.build as build_task
  File "/usr/local/lib/python3.10/site-packages/dbt/task/build.py", line 1, in <module>
    from .run import RunTask, ModelRunner as run_model_runner
  File "/usr/local/lib/python3.10/site-packages/dbt/task/run.py", line 8, in <module>
    from .compile import CompileRunner, CompileTask
  File "/usr/local/lib/python3.10/site-packages/dbt/task/compile.py", line 4, in <module>
    from .runnable import GraphRunnableTask
  File "/usr/local/lib/python3.10/site-packages/dbt/task/runnable.py", line 11, in <module>
    from .printer import (
  File "/usr/local/lib/python3.10/site-packages/dbt/task/printer.py", line 22, in <module>
    from dbt.tracking import InvocationProcessor
  File "/usr/local/lib/python3.10/site-packages/dbt/tracking.py", line 26, in <module>
    import pytz
ModuleNotFoundError: No module named 'pytz'
The command '/bin/sh -c dbt deps' returned a non-zero code: 1
```

### Steps To Reproduce

Copy
```
FROM python:3.10
RUN pip install --upgrade pip
RUN pip install dbt-oracle==1.4.0
RUN dbt deps
```
into a file `Dockerfile` in an otherwise empty directory and run `docker build .`

### Environment

```markdown
- OS: Linux
- Python: 3.10
- dbt: 1.4.0
```

### Additional Context

This was actually a bug in dbt-core (https://github.com/dbt-labs/dbt-core/issues/7075), which is fixed in dbt-core 1.4.4.

Suggestion: Can you loosen the hard dependency on dbt-core 1.4.0 or replace it by dbt-core 1.4.4?

**Comment by aosingh** [comment]
@heiderich 

Thanks for reporting this. Could you let me know how is this `pytz` module installed ?

Meanwhile I will check the dbt-core version and plan a fix.

**Comment by heiderich** [comment]
@aosingh Using dbt-core 1.4.4, it should automatically be installed as a dependency thanks to https://github.com/dbt-labs/dbt-core/pull/7077/

I think guess #76 fixes the issue. Thanks for quickly taking care of it!

**Comment by aosingh** [comment]
@heiderich 

This should be fixed if you upgrade to `dbt-oracle==1.4.1`

Thanks for reporting this. Let me know if you have any questions.

**Comment by heiderich** [comment]
Thanks for the quick fix @aosingh. `dbt-oracle==1.4.1` fixes the issue.
